### PR TITLE
Link the navbar logo to homepage

### LINKED ISSUE #334: Add a link to image in the navbar
**Is your feature request related to a problem? Please describe.**
When I am clicking on the image in the nav bar it does not take me to a new webpage(web page is sack.edu.k)

**Describe the solution you'd like**
I like to add a link to this image.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by VishwaAmarasinghe**
## Purpose

The purpose of this PR is to fix #334 

## Goals
I add a anchor tagthat contains our website link.
### Screenshots
image

  
### Preview Link
https://vishwaamarasinghe.github.io/sack-site/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: _includes/static/navbar.html [hunk 1]
```diff
@@ -4,7 +4,10 @@
     <div class="col">
       <div class="row justify-content-start">
         <div class="col-sm-4">
-          <img class="nav-img" src="{{ site.baseurl }}/assets/img/nav-img.svg">
+          <a href="https://www.sack.edu.lk/#gsc.tab=0">
```
Comment (Piumal1999) [inline _includes/static/navbar.html:7]:
```suggestion
          <a href="../../">
```

@VishwaAmarasinghe We can directly use the relative path here.

Comment (VishwaAmarasinghe) [reply _includes/static/navbar.html]: Ok brother I updated that

Comment (jayasanka-sack) [reply _includes/static/navbar.html]: I think it should be `<a href="{{ site.baseurl }}">`

Comment (Piumal1999) [reply _includes/static/navbar.html]: @VishwaAmarasinghe could you please the above suggestion

Comment (VishwaAmarasinghe) [reply _includes/static/navbar.html]: Ok aiyya

**Comment by VishwaAmarasinghe** [comment]
I did that

**Review by Piumal1999 — APPROVED** [review]
Awesome! Thank you @VishwaAmarasinghe

**Pull request merged**